pkg/utils/k8sutil.go

```diff
@@ -1058,3 +1073,19 @@ func addInitContainerAnnotation(dpm *v1beta1.Deployment) error {
 	}
 	return nil
 }
+
+//IsKubernetesResourceNotFoundError returns true if resource is not found
+func IsKubernetesResourceNotFoundError(err error) bool {
```

Comment:
Don't create functions that just call another function - it adds cognitive overhead for no benefit.  Just use `k8sErrors.IsNotFound` directly.